### ISSUE TITLE
Mastodon

### ISSUE THREAD

**Issue by FrittenLoverMarvin** [webinstall/webi-roadmap#11]
What do you think about using [Mastodon](https://joinmastodon.org/) instead of [Twitter](https://twitter.com/_webinstall)?

**Comment by coolaj86** [comment]
@FrittenLoverMarvin I don't think it has to be an "instead of", but I wouldn't really know which server to join. However, if it extends audience reach, all the better.

What are your suggestions?

**Comment by FrittenLoverMarvin** [comment]
> @FrittenLoverMarvin I don't think it has to be an "instead of", but I wouldn't really know which server to join. However, if it extends audience reach, all the better.
> 
> What are your suggestions?

[Fosstodon](https://fosstodon.org/) would be fine, @coolaj86.
